### PR TITLE
Fix session undefined bug

### PR DESCRIPTION
**Pull request by vineetmimrot**
After the Azure Login, from the investigation I found that it takes time to update the subscription list. So I have added timeout for the same.

The bug is reproducible by:
1. Do Azure signout
2. Remove Deploy to Azure and Azure Account extension.
3. Install Deploy to Azure extension.
4. Try to setup pipeline, for the first time you will hit "session" is undefined. Though from second time onwards things work fine.

### CODE REVIEW

**Comment by vineetmimrot** [comment]
https://github.com/microsoft/vscode-deploy-azure/issues/137

**Comment by vineetmimrot** [comment]
Raised issue
https://github.com/microsoft/vscode-azure-account/issues/210

File: src/configure/configure.ts [hunk 1]
```diff
@@ -40,12 +40,13 @@ export async function configurePipeline(node: AzureTreeItem) {
...
                 telemetryHelper.setTelemetry(TelemetryKeys.AzureLoginRequired, 'true');
 
-                let loginOption = await vscode.window.showInformationMessage(Messages.azureLoginRequired, Messages.signInLabel, Messages.signUpLabel);
+                const loginOption = await vscode.window.showInformationMessage(Messages.azureLoginRequired, Messages.signInLabel, Messages.signUpLabel);
                 if (loginOption && loginOption.toLowerCase() === Messages.signInLabel.toLowerCase()) {
                     telemetryHelper.setTelemetry(TelemetryKeys.AzureLoginOption, 'SignIn');
                     await vscode.window.withProgress({ location: vscode.ProgressLocation.Notification, title: Messages.waitForAzureSignIn },
                         async () => {
                             await vscode.commands.executeCommand("azure-account.login");
+                            await extensionVariables.azureAccountExtensionApi.waitForSubscriptions();
```
Comment (kanika1894) [inline src/configure/configure.ts:49]: So this is basically ensuring that we have the list of subscriptions with us and don't run into the error, right?

Comment (vineetmimrot) [reply src/configure/configure.ts:49]: yes.

**Pull request merged**